.ibm/pipelines/resources/config_map/configmap-app-config-rhdh.yaml

```diff
@@ -7,6 +7,15 @@ data:
7oE33ZE5Xc+CMRt4U+XYdrHI/QTfPf7KQ3/y3T0ILT4DdIzYy878qHGdNJ9quMMD41tNuvE7A/ZVEdNVQsREIFiEoBRPRzp/xHKyWNSOJH5JYzo00mHaj1Gbkz3+JPbf+1aM70SgdEA/9+W+AHgtmgNbRpr8DLW2Wf/oxJrk7vMmChMmjXMOTB3SIUQAREQgdwEYFB3QyguqHaHvgsdAkPK7euql9T7FJFUdtpgZwYSej2SxEqcCN8HzBc2cCV6HS4/V7OPYVu4WgUX/75qrZvqJQIiUHoCMJyvINcjoDdCuSKtCUNK0pGvTJlo3/HuGXjXK2FWtPBl+fgNpu/WcCVXDtVailWpZ1O7kmuksouACIhA+QhEvjJlVVq1xe8BncRr28pXswhyXotHZ/zusUaQfFmTwIW/rawFUOYiIAIiUEUEimJMpxzF34CZcVXEqeqqUteidv60t+ouniokAiIQOwJFMaaJWrrmltjVVgVKEvjT1BOcBckTHUVABERABAojUDRjOmu48x6KdkdhxVPsohBoZm4qSrpKVAREQARqlEDRjCl54umm39Yo19hWG9fk7plD7H94HdtCq2AiIAIiEHMCRTWmM4bhReGOVkFxagN4c9Q1cSqPyiICIiAC1UCgqMaUgPAehKtwwLuHJOUmgIfCfjdtqPN2ucuh/EVABESg2ggU3Zhi8Oa7SEdXG7gKrM8XeCnFmAost4osAiIgArEnUHRjSgJ4kTr/VWBW7GlUdwEvfm1EvP6Norpxq3YiIAK1RKAkxjQB1DUX1hLYmNX1KUxo9GR1zC6KiiMCIlA9BEpmTGcOd6YC29XVg65CauKaNXiC95wKKa2KKQIiIAIVSQDjbGkFf9HGF+H3K22uNZybY0bOHOro1YE13ARUdREQgeITKNnKNFkVZDgS3/H/LJISELhHhrQElJWFCIhAzRMouTGdPsx5A789PanmyRcbgGNmu0vNiGJno/RFQAREQASMKbkxJXSslibiL8Cu0wUoEgHeJ3XN0FlnOyuKlIOSFQEREAERSCFQFmPK/PFfmufj8HBKWfQ1OgK/xNun/hldckpJBERABEQgG4GyGVMW6rs15ji8lYcPJEmiI3Aunpz+S3TJKSUREAEREIFcBMpqTPESgdWt6swRKOSbuQoq/9wEMDEZi9+T6s8FcqNSCBEQARGIlEDJfxpjK33f8W5PxzGT4NfT5i+3AATwhwK4F31mgJAKIgIiIAIiEDGBsq5Mk3Xhf5/Cqh+Mc/4HqiR/AjfLkOYPTTFEQAREICoCsTCmrAz/rg1/D7Y/vr4RVeVqIR2s6K/H1u7ptVBX1VEEREAE4kogNsaUgKYPdea3XGP2xdcX4gosVuVyzaUzhjrnxapMKowIiIAI1CCBWNwztXHvN969By93+IXNT27G4HekJ80Y7owXCxEQAREQgfITiNXKNBUHft7xS5yPTnXT9wSBj/DU7r4ypGoNIiACIhAfArFdmSYR4cX4P8f3cdCNkm41fHyybo05edoI59MaZqCqi4AIiEDsCMTemJLYgHFu14bm5mZ8PSR2BEtVIMdcgid2rypVdspHBERABEQgOIGKMKbJ6vSb4J5uXHM9zlsk3ar9iHcYv4xt3XNeGu7oTVHVfrFVPxEQgYolUFHGlJQHTHB7rMWbflDwoyqWeoCCw4g24KdCl+Bp3bEBgiuICIiACIhAGQlUnDFNssIq9VCs2C5DBXZKulXLEXW6G78fHcOfClVLnVQPERABEahmAhVrTJMXBUb1ZGz98reWFf8qQhjQJ/CTl2vxn6/Tk/XTUQREQAREIP4EKt6YJhEnjKoxp8Gwbpt0q5Sj65hH69aam6ef5EyplDKrnCIgAiIgAj8QqBpjmqxS3zvdw1CpYTg/KOkW0+MyvJRiIgzp+FlDnNkxLaOKJQIiIAIiEIBA1RnTZJ13n+j2au6aY3Ff9Ui4bZN0L/cRDxZNwoNFD7Tobu6bspezptzlUf4iIAIiIAKFE6haY5qKps8EdyfcizwIr3vaH8asf6pfsb8jv6WAPBk6yW1hnph5grO42HkqfREQAREQgdISqAljmop0wK1ue7eV2QNufaG7QHeAweuQGqaQ70hrHuK/DrCv4DgT/+gyq5D0FFcEREAERCD+BGrOmNouSd+73M7N1pgtGpqZ7k6D6QKD2AlgOuBhpnZ4wrYNzlsiHnaNzSqscFfhXuc38FuC8y/g/gnue37YvMG836aVmfPMCc5yWx5yEwEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREAEREIGqI/D/AdkuDh1O2hHWAAAAAElFTkSuQmCC"
```

Comment:
Instead of this long text can we put the logo somewhere and use the path or address here?

Reply:
I thought the same, but according to [docs](https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.1/html/getting_started_with_red_hat_developer_hub/ref-additional-rhdh-customizations_rhdh-getting-started) an embedded base64 encoded image is expected.

Reply:
I can change to a different image that would make the string shorter (maybe an encoded svg), but it would still need to be placed there

Reply:
changed logos to shorter strings.